File: Java Script/Práticas/nota.js
function filtrar() {
    let aluno = document.getElementById("aluno").value;
    let nota1 = document.getElementById("nota1").value;
    let nota2 = document.getElementById("nota2").value;
    let nota3 = document.getElementById("nota3").value;
    let media = (nota1+nota2+nota3)/4
    if (media >=7) {
        let situacao= "aprovado"
        }
    else {
        let situacao= "reprovado"
    }

    let tabela=document.getElementById("tabela")
    let linha=tabela.insertRow()
    linha.insertCell(0).textContent=aluno
    linha.insertCell(1).textContent=nota1
    linha.insertCell(2).textContent=nota2
    linha.insertCell(3).textContent=nota3
    linha.insertCell(4).textContent=situacao
}

File: C++/nota.cpp
#include <iostream>
using namespace std;

main() {
    cout << "Situacao do aluno" << endl;
    string aluno;
    cout << "Nome do Aluno: ";
    cin >> aluno;

    float n1;
    float n2;
    float n3;
    
    cout << "Nota do 1B" << endl;
    cin >> n1;
    cout << "Nota do 2B" << endl;
    cin >> n2;
    cout << "Nota do 3B" << endl;
    cin >> n3;

    float media;
    media=(n1+n2+n3)/3;

    if (media>=7) {
        cout << aluno << " Aprovado(a)" << endl;
        cout << "Media: " << media << endl;
    }
    else {
        cout << aluno << " Reprovado(a)" << endl;
        cout << "Media: "<< media << endl;
    }
    return 0;
}